README.md

```diff
@@ -50,3 +69,10 @@ Head to the address it gives you to check if your changes have worked. More info
   * Removed kdb+ tick code
   * Moved KDBBASEPORT assignment to setenv.sh
   * Feed process uses timer library
+- **1.4.0, January 2018**:
```

Comment:
This should be version 1.5.0

With 1.4.0 this release note wasn't amended.  Can you modify this to 1.5.0 and add in a note for 1.4.0.  You also need to modify appconfig/application.txt